samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages_ca.properties

```
@@ -619,3 +619,4 @@ seb_download_configuration=Descarregar Configuraci\u00f3
 seb_download_seb=Descarregar Safe Exam Browser
 seb_launch_seb=Executar Safe Exam Browser
 seb_quit_browser=Eixir de Safe Exam Browser
+seb_preview_info=Assessments that are configured to use Safe Exam Browser can not be previewed.
```

@bgarciaentornos Is that correct?
```suggestion
seb_preview_info=Les proves que estan configurades per utilitzar Safe Exam Browser no es poden previsualitzar.
```

---

Yep. I see "prova" is being used elsewhere but begin_assessment_msg_seb_required says 'examen', should we change it?

---

Yeah, I think `begin_assessment_msg_seb_required` should be changed, that is the only string that uses "examen".